test/unit_test.py

```diff
@@ -145,7 +145,7 @@ def test_upstream_from_config(self):
         upstream_base = read_upstream_from_spack_yaml(
             os.path.join(os.path.normpath(spack_c2sm_path),
                          'upstreams/daint/base'))
-        self.assertEqual('/project/g110/spack/upstream/daint_v0.20.1.0/base',
```

Comment:
This change is not needed, we only check upstream "base", its path remained unchanged. So revert.

Reply:
done